docs/plugins.md

```diff
@@ -0,0 +1,299 @@
+module.exports = {
+  getUsers: async(params) => {
+    return User.fetchAll(params);
+  }
+}
+```
+
+Usage from the plugin:
+```js
+// ./plugins/my-plugin/api/controllers/index.js
+module.exports = {
+  getUsers: async() => {
+    // Get parameters from the request
+    const {limit, sort} = ctx.request.query;
+
+    // Get the list of users using the plugins queries
+    const users = await strapi.plugins.myPlugin.config.queries.getUsers({limit, sort});
+
+    // Send the list of users as response
+    ctx.body = users;
+  }
+}
+```
+
+### Plugin admin interface development
+
+#### Introduction
+
+Strapi admin panel and plugins attend to be an easy and powerful way to create new features usable from the admin panel.
+
+The admin panel is a [React](https://facebook.github.io/react/) application which can embed other React applications. These other React applications are actually the `admin` parts of each Strapi plugins.
+
+#### Start the project in development mode
+
+To start the project in development mode:
+ - Open a new tab in you terminal
+ - Move to the `./plugins/my-plugin` folder
+ - For the moment `npm link strapi-helper-plugin`
+ - Install the dependencies: `npm install`
+ - Start the project: `npm start`
+ - Open the admin panel in your web browser: [http://localhost:1337](http://localhost:1337)
+ - You should now be able to see the plugin in development mode
+
+Explanations: by doing the steps above, you start a Webpack (to build the assets) and an Express server (exposing the assets, running on port `3000`). The admin panel automatically detects if a process is running on port `3000`. If it does, it load the `main.js` file which is, by default, the name of the build. This system enable hot reload to make developers life easier.
+
+Note: currently, you can develop only one plugin at a time.
+
+#### Folders and files structure
+
+The admin panel related part of each plugin is contained in `./plugins/my-plugin/admin` folder. Here is its following structure:
+ - `app`: source code directory
+   - `components`: contains the list of React components used by the plugin
+   - `containers`
+    - `App`: container used by every other containers
+    - `HomePage`
+      - `actions.js`: list of [Redux actions](http://redux.js.org/docs/basics/Actions.html) used by the current container
+      - `constants.js`: list of actions constants
+      - `index.js`: React component of the current container
+      - `messages.js`: list of messages for translations (optional)
+      - `reducer.js`: list of [Redux reducers](http://redux.js.org/docs/basics/Reducers.html) used by the current container
+      - `sagas.js`: list of [redux-sagas functions](https://github.com/redux-saga/redux-saga) used by the current container (optional)
+      - `selectors.js`: list of [selectors](https://github.com/reactjs/reselect) used by the current container
+      - `styles.scss`: style of the current container (optional)
+    - `AboutPage`: secondary page
```

Comment:
If we follow the alphabetical order it should be before the `App` and `HomePage` containers in the list